content/docs/product/managed-apps/dev-experience/git-subtree.md

```diff
@@ -409,4 +409,34 @@ In general, we have two options here:
    git commit -am "applied custom changes"
    # you're ready continue with updating to the current state
    ```
+## Fixing when original subtree-split commit is lost
 
+For reasons we're not completely sure of (but that for sure include history rewrite with `git push --force`), it might happen that the commit ID that is merged as a subtree from the artificial sub-directory tree created with `git subtree split` might change its commit ID, even though the set of changes is still the same. One way you can get out of this situation is:
```

Comment:
"... the commit ID ... might change it's commit ID, ..." **?**

Maybe we can try to split this long sentence and clarify a little. The problem is that the targetted commit ID of the external subtree vanishes under certain circumstances, right?

Same should be true for `sub-modules`, not just for `sub-trees`, right?